bin/plugin/commands/build-plugins.js

```diff
@@ -0,0 +1,296 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs-extra' );
+const path = require( 'path' );
+const glob = require( 'fast-glob' );
+
+/**
+ * Internal dependencies
+ */
+const { log, formats } = require( '../lib/logger' );
+
+exports.options = [];
+
+/**
+ * Command for copying the files required for the standalone plugin.
+ */
+exports.handler = async () => {
+	const pluginsFile = path.join( '.', 'plugins.json' );
+	fs.readFile( pluginsFile, 'utf8', ( err, jsonString ) => {
+		if ( err ) {
+			log(
+				formats.error(
+					`\nError reading file from disk: "${ err }"\n\n`
+				)
+			);
+		}
+
+		try {
+			const modules = JSON.parse( jsonString );
+			for ( const key in modules ) {
+				const pluginKey = key;
+				const pluginVersion = modules[ pluginKey ]?.version;
+				const pluginSlug = modules[ pluginKey ]?.slug;
+
+				if ( ! pluginVersion || ! pluginSlug ) {
+					log(
+						formats.error(
+							'\nThe given module configuration is invalid, the module JSON object is missing the "version" and/or "slug" properties, or they are misspelled.\n\n'
+						)
+					);
+					return;
+				}
+
+				try {
+					// Copy module files from the folder.
+					const modulePath = path.join( '.', 'modules/' + key );
+					try {
+						fs.copySync( modulePath, './build/' + pluginSlug, { overwrite: true } );
```

Comment:
Instead of overwriting, I think we should rather delete the existing folder before. Otherwise, if we delete a file from a module, it may still remain in the built plugin. Overwriting alone would not protect us from that I think.